File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int x=1;
        
        
        for(int i=1; i<nums.length;i++)
        {
            if (i==nums.length-1)
            {
                nums[x++]=nums[i];
                break;
            }
                
            if(nums[i-1]!=nums[i+1])
                nums[x++]=nums[i];
            
        }
        return x;
    }
}

File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.c
int removeDuplicates(int* nums, int numsSize) {
    int x = 1;
    for(int i=1; i<numsSize;i++)
    {
        if (i==numsSize-1)
            {
                nums[x++]=nums[i];
                break;
            }
        if(nums[i-1]!=nums[i+1])
            nums[x++]=nums[i];
    }
   
    return x;
}